src/viam/sdk/CMakeLists.txt

```diff
@@ -202,6 +202,11 @@ target_link_libraries(viamsdk
 )
 
 
+if (APPLE)
```

Comment:
@abe-winter - I was out on PTO when this review happened, but this looks a little questionable to me. Can you explain why it was required as part of this change?